composer.json

```diff
@@ -22,13 +22,13 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "laminas/laminas-stdlib": "^2.7 || ^3.0"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~2.2.1",
-        "laminas/laminas-mail": "^2.12",
-        "phpunit/phpunit": "^9.5"
+        "laminas/laminas-coding-standard": "~2.4.0",
+        "laminas/laminas-mail": "~2.19.0",
```

Comment:
Hmm, this bi-directional dependency is problematic - not sure how to solve this, other than ignoring platform requirements during installation

Reply:
@gsteel you dealt with a bunch of these recently: have any suggestions?

Reply:
We'll have to checkout `mime` as `2.99.x` in `pre-install.sh` like this:
https://github.com/laminas/laminas-view/blob/2.25.x/.laminas-ci/pre-install.sh

Reply:
i got the same error, did i miss something ?

```
- git co -b 2.99.x
Switched to a new branch '2.99.x'
- composer update
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Problem 1
- Root composer.json requires laminas/laminas-mail ~2.19.0 -> satisfiable by laminas/laminas-mail[2.19.0].
    - laminas/laminas-mail 2.19.0 requires laminas/laminas-mime ^2.10.0 -> satisfiable by laminas/laminas-mime[2.10.0] from composer repo (https://repo.packagist.org) but laminas/laminas-mime is the root package and cannot be modified. See https://getcomposer.org/dep-on-root for details and assistance.

```

Reply:
Can you try pushing that? It may be composer trying to require a stable version in your environment...

Reply:
@Ocramius seem to be good, thanks @gsteel !

Reply:
@fezfez this should be `^2.19.0` BTW: `~2.19.0` is too restrictive

Reply:
done